commands/update_organization_setcredentials.go

```diff
@@ -0,0 +1,351 @@
+package commands
+
+import (
+	"fmt"
+	"net/http"
+	"os"
+	"strings"
+
+	"github.com/fatih/color"
+	"github.com/giantswarm/gsclientgen/models"
+	"github.com/giantswarm/gsctl/client/clienterror"
+	"github.com/giantswarm/gsctl/config"
+	"github.com/giantswarm/microerror"
+	"github.com/spf13/cobra"
+)
+
+const (
+	updateOrgSetCredentialsActivityName = "set-org-credentials"
+)
+
+var (
+
+	// UpdateOrgSetCredentialsCommand performs the "update organizatio set-credentials" function
+	UpdateOrgSetCredentialsCommand = &cobra.Command{
+		Use:     "set-credentials",
+		Aliases: []string{"sc"},
+		Short:   "Set credentials of an organization",
+		Long: `Set the credentials used to create and operate the clusters of an organization.
+
+Setting credentials of an organization will result in all future cluster
+being run in the account/subscription referenced by the credentials. Once
+credentials are set for an organization, this cannot be undone.
+
+For details on how to prepare the account/subscription, consult the documentation at
+
+  - https://docs.giantswarm.io/guides/prepare-aws-account-for-tenant-clusters/ (AWS)
+  - https://docs.giantswarm.io/guides/prepare-azure-subscription-for-tenant-clusters/ (Azure)
+
+`,
+		Example: `
+  gsctl update organization set-credentials -o acme \
+    --aws-operator-role arn:aws:iam::<AWS-ACCOUNT-ID>:role/GiantSwarmAWSOperator \
+    --aws-admin-role arn:aws:iam::<AWS-ACCOUNT-ID>:role/GiantSwarmAdmin
+
+  gsctl update organization set-credentials -o acme \
+    --azure-subscription-id <AZURE-SUBSCRIPTION-ID> \
+    --azure-tenant-id <AZURE-TENANT-ID> \
+    --azure-client-id <AZURE-CLIENT-ID> \
+    --azure-secret-key <AZURE-SECRET-KEY>
+`,
+
+		// PreRun checks a few general things, like authentication and flags
+		// plausibility.
```

Comment:
`plausibility` isn't the right term here. `compatibility` is probably more correct